### DISCUSSION TITLE
maven projects problem - "Error while storing the mojo status"

### DISCUSSION THREAD

**Discussion by ChrlB** [apache/netbeans#7817] [General]
### Apache NetBeans version

Apache NetBeans 22

### What happened

Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.11.0:compile (default-compile) on project mavenproject1: Error while storing the mojo status: C:\Users\user\OneDrive\Documents\NetBeansProjects\mavenproject1\target\maven-status\maven-compiler-plugin\compile\default-compile\inputFiles.lst -> [Help 1]  To see the full stack trace of the errors, re-run Maven with the -e switch. Re-run Maven using the -X switch to enable full debug logging.  For more information about the errors and possible solutions, please read the following articles: [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException

### How to reproduce

fixe it

### Did this work correctly in an earlier version?

Apache NetBeans 19

### Operating System

windows

### JDK

20

### Apache NetBeans packaging

Apache NetBeans provided installer

### Are you willing to submit a pull request?

Yes

**Comment by mbien** [comment]
sounds like a file permission issue to me. `C:\Users\user\OneDrive` is probably a synced folder which is another indication that it is likely access rights.
